### DISCUSSION TITLE
force alternate search filters

### DISCUSSION THREAD

**Discussion by castdrian** [ReVanced/revanced-discussions#158] [Question]
The android app sometimes lets you filter the search reults via a bar of buttons after you have searched, ie "Unwatched", "Short", "Long", "New", "Old".

I'm unclear on what causes this bar to appear or to not appear - it is very inconsistent-, but if someone figures that out I'm fairily certain you can force it to always appear.

Ideally being able to filter out watched videos by default or blacklist channels from appearing in the search results/home would be what I would want but I'm more or less certain that that is not quite possible/requires a lot of hacking around in saving chanelids and whatnot

**Comment by oSumAtrIX** [comment]
Can you send a picture of the issue you mentioned?

**Comment by castdrian** [comment]
![Screenshot_20220414-103605.jpg](https://user-images.githubusercontent.com/22133246/163347105-0f0c46b9-6be1-462a-88ad-04f6139a8a16.jpg)![Screenshot_20220414-103900.jpg](https://user-images.githubusercontent.com/22133246/163347635-49b95fac-a7b7-4920-8bf5-a530de8ddf0e.jpg)
